agera/src/main/java/com/google/android/agera/Observables.java

```diff
@@ -45,17 +43,23 @@
    */
   @NonNull
   public static Observable compositeObservable(@NonNull final Observable... observables) {
+    return compositeObservable(0, observables);
+  }
+
+  @NonNull
+  static Observable compositeObservable(final int shortestUpdateWindowMillis,
+      @NonNull final Observable... observables) {
     if (observables.length == 0) {
-      return new CompositeObservable();
+      return new CompositeObservable(0);
```

Comment:
`CompositeObservable` doubles as a null observable. Even if it's not doing anything it still needs to follow the contract of throwing exceptions for non paired calls etc (purely a safety precaution) so in the end an empty `CompositeObservable` will look exactly the same